it/controllers/employment/EmployerStartDateControllerISpec.scala

```diff
@@ -41,8 +41,6 @@ class EmployerStartDateControllerISpec extends IntegrationTest with ViewHelpers
 
   val charLimit: String = "ukHzoBYHkKGGk2V5iuYgS137gN7EB7LRw3uD0vujYg00ZtHwo3so0yOOCEoAK9vuPiP374QKOe0o"
 
-  private val userRequest = User(mtditid, None, nino, sessionId, affinityGroup)(fakeRequest)
-
   private def employmentUserData(isPrior: Boolean, employmentCyaModel: EmploymentCYAModel): EmploymentUserData =
```

Comment:
can you Incorporate your builder into these employmentUserData methods? I think this applies for all the employment specs

Reply:
Done